Periodic_4_hyperbolic_triangulation_2/doc/Periodic_4_hyperbolic_triangulation_2/Periodic_4_hyperbolic_triangulation_2.txt

```diff
@@ -17,9 +17,7 @@ namespace CGAL {
 
 
 
-This package enables the computation of Delaunay triangulations of the Bolza
-surface, which is the most symmetric surface of genus 2. The Bolza surface is
-a hyperbolic closed compact orientable surface.
+This package enables the computation of Delaunay triangulations of the Bolza surface, which is the most symmetric surface of genus 2. The Bolza surface is a hyperbolic closed compact orientable surface. For triangulations on general hyperbolic surfaces, we refer to the package \ref Chapter_Hyperbolic_Surface_Triangulations "Hyperbolic Surface Triangulations".
```

Comment:
Probably need to add `Hyperbolic_surface_triangulation_2` to the dependencies of `Periodic_4_hyperbolic_triangulation_2` to resolve the documentation warning:
```
/home/cgal-testsuite/cgal_doc_build/CGAL-6.1-Ic-65/doc/Periodic_4_hyperbolic_triangulation_2/Periodic_4_hyperbolic_triangulation_2.txt:20: warning: unable to resolve reference to 'Chapter_Hyperbolic_Surface_Triangulations' for \ref command
```